assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -566,6 +459,88 @@ const ExamplePage = () => {
           </Paragraph>
         </Box>
 
+        <CodeSnippet
+          accountId={this.state.accountId}
```

Comment:
nit - we can remove `this.state` from all of these prop values since we destructure the values above 👍

Reply:
done